tests/conftest.py

```diff
@@ -1230,9 +1234,12 @@ def setup_for_snippet_external(
         install_mode: Optional[InstallMode] = None,
         relation_precedence_rules: Optional[List[RelationPrecedenceRule]] = None,
         use_pip_config_file: bool = False,
+        index_url: Optional[str] = None,
```

Comment:
We could make this backward compatible be translating the old options into the new ones, I don't know if we have to?  (I did and then removed it to force a cleanup of all testcases)

Reply:
I like it like this.